SharedConfig.targets

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <DevVars>DevVars.targets</DevVars>
     <DevVarsLoc>$(SolutionDir)\$(DevVars)</DevVarsLoc>
-    <SubnauticaDir Condition="!Exists($(DevVarsLoc))">C:\Program Files (x86)\Steam\steamapps\common\Subnautica</SubnauticaDir>
+    <SubnauticaDir Condition="!Exists($(DevVarsLoc))">D:\Steam Library\steamapps\common\Subnautica</SubnauticaDir>
```

Comment:
Please don't change the default location; for this you can rename `DevVars.targets.example` to `DevVars.targets` and specify the location there.